index.html

```diff
@@ -42,6 +42,8 @@
       <section id="main_content">
         <div class="button">
           <a href="http://ginnivan.blob.core.windows.net/carnac/Carnac.application">Download</a>
+          <p class="top">click to begin</p>
```

Comment:
I really like this animation, but I'm not quite sure these words are necessary. What if we had something else here? 

One idea that comes to mind: we mention this is for Windows, but not which platforms it supports. We could say "requires Windows 7 or later" here instead..